cli/command/volume/create.go

```diff
@@ -50,7 +49,7 @@ func newCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	return cmd
 }
 
-func runCreate(dockerCli *command.DockerCli, opts createOptions) error {
+func runCreate(dockerCli command.Cli, opts createOptions) error {
```

Comment:
`command.DockerCli` is a struct, `command.Cli` is an interface. I don't want a pointer on the interface, I just want something that satisfy my interface (and `*command.DockerCli` is one of them)